resources/definitions/twotrees_sapphirepro.def.json

```
@@ -0,0 +1,38 @@
+{
+    "version": 2,
+    "name": "TwoTrees Sapphire Pro",
+    "inherits": "fdmprinter",
+    "metadata":
+    {
+        "visible": true,
+        "author": "MLoewe",
+        "manufacturer": "TwoTrees",
+        "file_formats": "text/x-gcode",
+        "platform": "twotrees_sapphirepro.stl",
+        "platform_texture": "twotrees_sapphirepro.png",
+        "platform_offset": [0, 0, 0],
+        "machine_extruder_trains":
+		{
+			"0": "twotrees_sapphirepro_extruder_0",
+			"1": "twotrees_sapphirepro_extruder_1"
+		}
```

Mix of tabs and spaces.

```suggestion
        "machine_extruder_trains":
        {
            "0": "twotrees_sapphirepro_extruder_0",
            "1": "twotrees_sapphirepro_extruder_1"
        }
```